pkg/crc/network/proxy_test.go

```diff
@@ -15,7 +15,7 @@ func TestValidateProxyURL(t *testing.T) {
 }
 
 func TestHidePassword(t *testing.T) {
-	url, err := URIStringForDisplay("https://user:secret@proxy.org:123")
+	url, err := hidePassword("https://user:secret@proxy.org:123")
```

Comment:
should this change be part of this commit, it should only have stringer change.

Reply:
I added more details in the commit message. Adding Stringer remove the need to have `URIStringForDisplay` public. I renamed it to be more in line with what it does.